recipes/muparser/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+sh ./configure --prefix=$PREFIX --disable-samples
+make -j2
```

Comment:
I had the impression that Travis CI had a limit of 1.5 (kind of weird but that's what it was). Though I believe I saw that for Linux so I don't remember what they do on Mac.